src/Engine/Renderer/Camera/Camera.cpp

```diff
@@ -43,7 +43,7 @@ void Camera::initialize() {
     m->addData( Mesh::VERTEX_COLOR, c );
 
     // Create the RO
-    m_RO = RenderObject::createRenderObject( m_name + "_RO", this, RenderObjectType::Fancy, m );
+    m_RO = RenderObject::createRenderObject( m_name + "_RO", this, RenderObjectType::Geometry, m );

```

Comment:
Set the ype of the Ro as ``RenderObjectType::UI`` or ``RenderObjectType::Debug``
Camera geometrical representation (which, even if usefull,  could be improved) must not be part of the scene but considered as ui  or debug object. This will allow to easily toggle the display of the camera in the scene throug main-app but also to facilitate some scene wide computation such as bounding boxes computation or colision detection, made only versus Geometry objects.